ncl/ncl_entries/dewtemp.ipynb

```diff
@@ -0,0 +1,123 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# dewtemp_trh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "NCL's [`dewtemp_trh`](https://www.ncl.ucar.edu/Document/Functions/Built-in/dewtemp_trh.shtml) calculates the dew point temperature given temperature and relative humidity using the equations from John Dutton's _\"Ceaseless Wind\"_ (pg. 273-274) and returns a temperature in Kelvin\n",
+    "\n",
+    ":::{important}\n",
+    "To convert from Kelvin to Celsius `-273.15` and to convert from Celsius to Kelvin `+273.15`\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grab and Go"
```

Comment:
Just wondering if we want to keep the "Grab and Go" label if all of the code is within this section. I interpreted it more as a way to differentiate the fast code from the minutiae - but this notebook is very streamlined.

Reply:
That's a good point, I think it is currently worth keeping for consistency with the templates and in this case, I was planning on seeing if any other meteorology functions were short enough to put in one notebook with this one (i.e. daylight)